python/cudf/cudf/tests/test_rolling.py

```diff
@@ -536,10 +536,9 @@ def get_window_bounds(
     "indexer",
     [
         pd.api.indexers.FixedForwardWindowIndexer(window_size=2),
-        pd.core.window.expanding.ExpandingIndexer(),
-        pd.core.window.indexers.FixedWindowIndexer(window_size=3)
-        if PANDAS_LT_140
-        else pd.core.indexers.objects.FixedWindowIndexer(window_size=3),
+        pd.api.indexers.VariableOffsetWindowIndexer(
+            index=pd.date_range("2020", periods=5), offset=pd.offsets.BDay(1)
+        ),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏This _seems_ like it is testing something different? What am I missing? (I don't know a lot about the rolling indexers)

Reply:
This is testing _slightly_ different but using a more public API.

Before I presume this was meant to test that rolling was compatible with rolling window indexer objects, and this test was testing private indexer objects (used in `df.expanding` for example). I switched this to test the public indexer objects